### PR TITLE
Add compression level option for RocksDB checkpoint

### PR DESCRIPTION
**Pull request by tcrain**
### Why are the changes needed?

Allows the user to specify the compression level of the RocksDB checkpoint for both single and multithreaded compression.

Fixes an issue where the compression level for the multithreaded compression was not being used.

### Does this PR introduce any user facing changes?

Adds the property key `alluxio.master.metastore.rocks.checkpoint.compression.level`.

Removes the property key `alluxio.master.metastore.rocks.parallel.backup.compression.level` as its value was not being used.

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java [hunk 1]
```diff
@@ -208,18 +208,18 @@ public synchronized void writeToCheckpoint(OutputStream output)
       throw new IOException(e);
     }
 
+    int compressLevel = Configuration.getInt(
```
Comment (jenoudet) [inline core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java:211]: Do you anticipate needing to change the compression level while the master is running? If so, keep it this way, if not, make it a member variable.

**Review by jenoudet — APPROVED** [review]

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Comment by yyongycy** [comment]
alluxio-bot, cherry-pick this to branch-2.8 please

**Comment by yyongycy** [comment]
alluxio-bot, cherry-pick this to branch-2.9 please

**Pull request merged**